### DISCUSSION TITLE
How to set OMP,INTRA& INTER parameters?

### DISCUSSION THREAD

**Discussion by gdbhcxhmjk-z** [deepmodeling/dpgen#728] [Q&A]
As the guide in deepmd says: ‘Note that OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS should be carefully adjusted to achieve the best performance.’
Where should these parameters be set in dpgen ? In machine.json or param.json? And how to write it?

**Comment by njzjz** [comment]
These are environmental variables.
